media/js/firefox/os/firefox-os-2.5.js

```diff
@@ -8,6 +8,7 @@
     function trackClick(e) {
         e.preventDefault();
         var id = $(e.target).data('id');
+        var pixelId = $(e.target).attr('id');
```

Comment:
Seeing as we now need two attributes, we could query the dom once and cache the result e.g.:

    var $elm = $(e.target);
    var id = $elm.data('id');
    var pixel = $elm.data('pixel');